### PR TITLE
Issue/invalid reset password link

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: core/i18n/resources/en.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ Do you want to cancel them?`,
     errorMessage_plural: 'Error messages',
     formContainsErrors: 'Form contains errors',
     formContainsErrorsCannotSave: 'The form contains errors. Please, fix them before saving.',
+    goToHomePage: 'Go to home page',
```
Comment (minotogna) [inline core/i18n/resources/en.js]: is this used?

File: core/i18n/resources/en.js [hunk 2]
```diff
@@ -794,14 +795,15 @@ $t(common.cantUndoWarning)`,
...
+    temporaryMsg: '<p>This link is only valid for the next 7 days.</p>',
     userInvite: {
       subject: 'You have been invited to OpenForis Arena!',
       body: `<p>Hello,</p>
              <p>You have been invited to join the survey <strong>{{surveyLabel}}</strong> as {{groupLabel}}</p>
-             <p><a href="{{serverUrl}}">Click here to access OpenForis Arena</a></p>
-             {{temporaryMsg}}
+             <p><a href="{{urlResetPassword}}">Click here to access OpenForis Arena</a></p>
+             $t(emails.temporaryMsg)
+             <p>In the future, after you accept the invitation and assign a password to your account, you will be able to access OpenForis Arena using this <a href="{{urlServer}}">link</a></p>
```
Comment (minotogna) [inline core/i18n/resources/en.js]: Use the old message

File: webapp/guest/components/notLoggedInView.scss [hunk 3]
```diff
@@ -117,3 +117,9 @@
   left: 35vw;
   top: 15vh;
 }
+
+.not-logged-in__btn-go-to-home-page {
```
Comment (minotogna) [inline webapp/guest/components/notLoggedInView.scss]: This is not used

File: webapp/guest/resetPassword/useResetPasswordState.js [hunk 4]
```diff
@@ -37,14 +40,18 @@ export const useResetPasswordState = () => {
   }, [name, password, passwordConfirm])
 
   useOnUpdate(() => {
-    history.push(appModuleUri(appModules.home))
+    history.push(appModuleUri())
```
Comment (minotogna) [inline webapp/guest/resetPassword/useResetPasswordState.js]: why not home directly ?

File: webapp/guest/resetPassword/useResetPasswordState.js [hunk 5]
```diff
@@ -37,14 +40,18 @@ export const useResetPasswordState = () => {
...
   }, [resetComplete])
 
   // Check that reset password uuid is valid (user exists)
   useOnUpdate(() => {
-    if (user) setName(User.getName(user))
-    else setError('resetPasswordView.forgotPasswordLinkInvalid')
+    if (user) {
+      setName(User.getName(user))
+    } else {
+      history.push(appModuleUri())
```
Comment (minotogna) [inline webapp/guest/resetPassword/useResetPasswordState.js]: why not login directly?

File: core/i18n/resources/en.js (hunk 1 above)
Comment (SteRiccio) [reply core/i18n/resources/en.js]: no, right

File: core/i18n/resources/en.js (hunk 2 above)
Comment (SteRiccio) [reply core/i18n/resources/en.js]: ok

File: webapp/guest/components/notLoggedInView.scss (hunk 3 above)
Comment (SteRiccio) [reply webapp/guest/components/notLoggedInView.scss]: right

File: webapp/guest/resetPassword/useResetPasswordState.js (hunk 4 above)
Comment (SteRiccio) [reply webapp/guest/resetPassword/useResetPasswordState.js]: appModules.home is the default

File: webapp/guest/resetPassword/useResetPasswordState.js (hunk 5 above)
Comment (SteRiccio) [reply webapp/guest/resetPassword/useResetPasswordState.js]: you can be logged in already

**Pull request merged**